src/main/java/elements/PictureImpl.java

```diff
@@ -0,0 +1,36 @@
+package elements;
+
+
+import org.openqa.selenium.ElementNotInteractableException;
+import org.openqa.selenium.ElementNotVisibleException;
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+/***
+ * Wrapper class with Picture functionalities
+ */
+public class PictureImpl extends ElementImpl implements Picture {
+
+    public PictureImpl(WebElement element) {
+        super(element);
+    }
+
+    @Override
+    public boolean isDisplayed() {
+        try {
+            getWrappedElement().isDisplayed();
+            return true;
```

Comment:
why do you return true here? why not getWrappedElement().isDisplayed(); ?

Reply:
what if the element is not displayed? your method will return true anyway

Reply:
Also what is the return statement of isDisplayed() ? that's boolean, it means you can use it in return

Reply:
fixed